assets/js/state/sagas/clusters.test.js

```diff
@@ -0,0 +1,19 @@
+import { recordSaga } from '@lib/test-utils';
+import { clusterDeregistered } from '@state/sagas/clusters';
+import { removeCluster } from '@state/clusters';
+
+describe('Clusters sagas', () => {
+  it('should trigger reducer to remove cluster', async () => {
```

Comment:
Maybe `should remove the cluster` simply?

Reply:
This test, is testing that the saga dispatches the correct action to the reducer. i.e. it does not validate the state change (because we test this in the reducer test).

Reply:
But that's going technical...
At the end this saga aims to update our state internally, in our domain terminology this is to `remove a cluster`.
Anyway, I don't want to force anything, it is just my point of view

Reply:
Okay no worries, changed it. Was just making sure we are on the same page 👍